docs/user-guide/src/bmo/advanced_instance_customization.md

```diff
@@ -0,0 +1,55 @@
+# Instance Customization
+
+Below we cover more advanced instance customization, more complex use-cases
+and/or where customization of the metal3 deployment may be required.
+
+For more general guidance around instance customization refer to the
+[instance customization](./instance_customization.md) section.
+
+## Pre-Provisioning NetworkData
+
+*Pre-provisioning network data* describes the desired networking configuration for the
+deploy ramdisk (IPA). Usage of this API requires an IPA ramdisk image with a tool
+capable of interpreting and applying the data such as *cloud-init*, *Glean* or alternative
+*(note the default community supported ramdisk does not currently contain such a tool)*.
+
```

Comment:
WDYY, should we add an extra sentence or paragraph pointing to a custom IPA build example that can build an IPA this functionality e.g.:
https://github.com/metal3-io/project-infra/blob/main/jenkins/scripts/dynamic_worker_workflow/build_ipa.sh

Or mybe point out that simple-init is the upstream element that can provide this functionality during IPA building?

Reply:
Good point, I think it would be best to reference the upstream simple-init element and IPA builder, since the project-infra scripts may be subject to change in future?

Reply:
Done